### PR TITLE
Fix issues found by go lint or go vet within the build code

### PR DESCRIPTION
**Pull request by csrwng**

### CODE REVIEW

**Comment by csrwng** [comment]
@bparees ptal

File: pkg/build/api/v1beta1/types.go [hunk 1]
```diff
@@ -202,7 +202,7 @@ type CustomBuildStrategy struct {
 
 	// Tag is the name of image stream tag to be used as the FROM image, it only
 	// applies when From is specified and the Kind of From is ImageStream.
-	Tag string `json:"tag,omitempty`
+	Tag string `json:"tag,omitempty"`
```
Comment (bparees) [inline pkg/build/api/v1beta1/types.go]: this change breaks the UI code right?  we need an update to consume it.

**Comment by bparees** [comment]
most of this looks fine, my only concern is the json annotation fixes which as i think we saw, alters the field name referenced by the javascript code (changes the case)

File: pkg/build/api/v1beta1/types.go (hunk 1 above)
Comment (csrwng) [inline pkg/build/api/v1beta1/types.go]: true... let me check

**Comment by csrwng** [comment]
@bparees - I changed the v1 types to use "Tag". Duration works ok, since it's still interpreted correctly.

**Comment by bparees** [comment]
ok.  i guess that's what we have to do for compatibility, but when we switch the UI to the v1b3 api we'll need to remember to fix the tag reference in the javascript :/

[merge]

**Pull request merged**